src/clients/api/queries/getVTokenBalancesAll.ts

```diff
@@ -0,0 +1,49 @@
+import { Contract } from 'web3-eth-contract';
+
+export interface IGetVTokenBalancesAllInput {
+  venusLensContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | null | undefined;
+  vtAddresses: string[];
+}
+
+interface IGetVTokenBalancesAllResponse extends Array<string> {
+  balanceOf: string;
+  balanceOfUnderlying: string;
+  borrowBalanceCurrent: string;
+  tokenAllowance: string;
+  tokenBalance: string;
+  vToken: string;
+}
+
+export interface IGetVTokenBalancesAllOutput {
+  balanceOf: string;
+  balanceOfUnderlying: string;
+  borrowBalanceCurrent: string;
+  tokenAllowance: string;
+  tokenBalance: string;
+  vToken: string;
+}
```

Comment:
Shall we use types instead of interfaces here too? This way we're consistent with the other queries.

Reply:
Ahh this is where I mean to post my other comment
> Interface is the correct option here. We are describing the shape of data/ object. Types are good for aliasing types of data (unions, primatives, other types ect.)